### PR TITLE
Delete chore and reward apis

### PR DESCRIPTION
**Pull request by dishadc**
Added endpoints and tests for delete chore and delete reward apis. Also needed to change insert queries for the tests.

### CODE REVIEW

File: Backend/db/queries.py [hunk 1]
```diff
@@ -238,3 +233,16 @@ def update_chore(columns: list) -> str:
...
         query = query.set(Field(column), Parameter("?"))
     return query.get_sql()
+
+
+##########################
+#  SQL DELETE statements #
+##########################
+
+
+def delete_chore():
```
Comment (phiphphi) [inline Backend/db/queries.py]: Can replace with PyPika equivalent:
```
def delete_chore() -> str:
    query = (
        Query.from_(chores_table)
        .delete()
        .where(chores_table.ChoreId == Parameter("?"))
        .where(chores_table.ParentGoogleAccountId == Parameter("?"))
    )
    return query.get_sql()
```

File: Backend/db/queries.py [hunk 2]
```diff
@@ -238,3 +233,16 @@ def update_chore(columns: list) -> str:
...
+    return """DELETE FROM Chores WHERE ChoreId = ? and ParentGoogleAccountId = ?"""
+
+
+def delete_reward():
```
Comment (phiphphi) [inline Backend/db/queries.py]: Can replace with PyPika equivalent:

```
def delete_reward() -> str:
    query = (
        Query.from_(rewards_table)
        .delete()
        .where(rewards_table.RewardId == Parameter("?"))
        .where(rewards_table.ParentGoogleAccountId == Parameter("?"))
    )
    return query.get_sql()
```

File: Backend/db/queries.py (hunk 1 above)
Comment (dishadc) [reply Backend/db/queries.py]: thanks, forgot to build these with pypika.

Comment (dishadc) [reply Backend/db/queries.py]: fixed

File: Backend/db/queries.py (hunk 2 above)
Comment (dishadc) [reply Backend/db/queries.py]: fixed

**Review by phiphphi — APPROVED** [review]
Looks great!

**Review by yaretzj — APPROVED** [review]
Nice!

**Pull request merged**